src/core_plugins/kibana/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "CORE-WELCOME_MESSAGE": "<strong>Kibana</strong> <small>is loading. Give me a moment here. I'm loading a whole bunch of code. Don't worry, all this good stuff will be cached up for next time!</small>",
```

Comment:
I don't feel great about allowing html in translation strings. A broken tag in a single translation could break an entire page and it provides a lot of opportunity for abuse. At the same time I can see translators needing some control over the appearance of their text.

@hickeyma @srl295 what's the best practice here? How do other translation systems handle this?